Add a note about enabling FIPS for US1-FED users (#36347)
DataDog/documentation: content/en/opentelemetry/setup/ddot_collector/install/kubernetes_daemonset.md

  - Replace `<CLUSTER_NAME>` with a name for your cluster.
  - Replace `<DATADOG_SITE>` with your [Datadog site][1]. Your site is {{< region-param key="dd_site" code="true" >}}. (Ensure the correct **DATADOG SITE** is selected on the right.)

> {{% site-region region="gov" %}}

Author: ```suggestion
{{% site-region region="gov,gov2" %}}
```

> <div class="alert alert-info">For US1-FED, also set <code>useFIPSAgent: true</code> under <code>spec.global</code> to use the FIPS-compliant Agent image. See <a href="/agent/configuration/fips-compliance/">FIPS compliance</a>.</div>

Author: ```suggestion
<div class="alert alert-info">For FED, also set <code>useFIPSAgent: true</code> under <code>spec.global</code> to use the FIPS-compliant Agent image. See <a href="/agent/configuration/fips-compliance/">FIPS compliance</a>.</div>
```

Set `<DATADOG_SITE>` to your [Datadog site][2]. Otherwise, it defaults to `datadoghq.com`, the US1 site.

{{% site-region region="gov" %}}

> <div class="alert alert-info">For US1-FED, also set <code>useFIPSAgent: true</code> at the root of your <code>datadog-values.yaml</code> to use the FIPS-compliant Agent image. See <a href="/agent/configuration/fips-compliance/">FIPS compliance</a>.</div>

Author: ```suggestion
{{% site-region region="gov,gov2" %}}
<div class="alert alert-info">For FED, also set <code>useFIPSAgent: true</code> at the root of your <code>datadog-values.yaml</code> to use the FIPS-compliant Agent image. See <a href="/agent/configuration/fips-compliance/">FIPS compliance</a>.</div>
```
